earthpy/spatial.py

```diff
@@ -229,10 +228,23 @@ def _stack_bands(sources, dest=None, write_raster=False):
     Parameters
     ----------
     sources : list of rasterio dataset objects
-        A list with paths to the bands you wish to stack. Objects
+        A list of rasterio dataset objects you wish to stack. Objects
         will be stacked in the order provided in this list.
-    dest : a rio.open writable object that will store raster data.
-    write_raster : a flag to decide to write out the raster.
+    dest : string (optional)
+        Path to the where the output raster containing the stacked
+        layers will be stored.
+    write_raster : bool (optional)
```

Comment:
I think this parameter was removed from this function? will you please have a closer look at the function and remove it / adjust parameters as needed?